src/Sulu/Bundle/CoreBundle/Command/WebspacesInitCommand.php

```diff
@@ -158,6 +159,11 @@ private function setBasicLocalizationProperties(
             $node->setProperty($this->properties->getName('nodeType'), Structure::NODE_TYPE_CONTENT);
         }
 
+        if (!$node->hasProperty($this->properties->getName('nodeType'))) {
+            // set title
+            $node->setProperty($this->properties->getName('title'), 'Sulu CMF');
```

Comment:
yes the title will be set for all localizations ... for me this title is not so important ... but if we want we could place a translation or we ask for each startpage localization? i wouldn´t use a locale dependend title therefor Sulu CMF seems to me a good middle way (= /cc @chirimoya @dantleech  what did you think?